kvbc/include/categorization/updates.h

```diff
@@ -235,6 +235,8 @@ struct Updates {
 
   const CategoryInput& categoryUpdates() const { return category_updates_; }
 
+  CategoryInput&& categoryUpdates() { return std::move(category_updates_); }
```

Comment:
who is the client of this method? is it safe to add it? do we need any flag to tell us that the object was already moved to prevent a double move?